--- a/components/window/src/native_buffer/native_buffer/native_buffer_ffi.rs
+++ b/components/window/src/native_buffer/native_buffer/native_buffer_ffi.rs
@@ -38,33 +38,26 @@
     }
 }
 impl OH_NativeBuffer_Usage {
+    /// CPU read buffer
     pub const NATIVEBUFFER_USAGE_CPU_READ: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(1);
-    /// < CPU read buffer */
+    /// CPU write memory
     pub const NATIVEBUFFER_USAGE_CPU_WRITE: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(2);
-    /// < CPU write memory */
+    /// Direct memory access (DMA) buffer
     pub const NATIVEBUFFER_USAGE_MEM_DMA: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(8);
-    /// < Direct memory access (DMA) buffer */
-    /// **
-    /// * MMZ with cache
-    /// * @since 20
-    /// */
-    #[cfg(feature = "api-20")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "api-20")))]
-    pub const NATIVEBUFFER_USAGE_MEM_MMZ_CACHE: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(32);
-    /// < Direct memory access (DMA) buffer */
-    /// **
-    /// * MMZ with cache
-    /// * @since 20
-    /// */
+    /// MMZ with cache
+    ///
+    /// Available since API-level: 20
     #[cfg(feature = "api-20")]
     #[cfg_attr(docsrs, doc(cfg(feature = "api-20")))]
-    pub const NATIVEBUFFER_USAGE_HW_RENDER: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(256);
-    /// < For GPU write case */
+    pub const NATIVEBUFFER_USAGE_MEM_MMZ_CACHE: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(32);
+    /// For GPU write case
+    pub const NATIVEBUFFER_USAGE_HW_RENDER: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(256);
+    /// For GPU read case
     pub const NATIVEBUFFER_USAGE_HW_TEXTURE: OH_NativeBuffer_Usage = OH_NativeBuffer_Usage(512);
-    /// < For GPU read case */
+    /// Often be mapped for direct CPU reads
     pub const NATIVEBUFFER_USAGE_CPU_READ_OFTEN: OH_NativeBuffer_Usage =
         OH_NativeBuffer_Usage(65536);
-    /// < Often be mapped for direct CPU reads */
+    /// 512 bytes alignment
     pub const NATIVEBUFFER_USAGE_ALIGNMENT_512: OH_NativeBuffer_Usage =
         OH_NativeBuffer_Usage(262144);
 }
